temporal/api/cloud/nexus/v1/message.proto

```diff
@@ -0,0 +1,114 @@
+syntax = "proto3";
+
+package temporal.api.cloud.nexus.v1;
+
+option go_package = "go.temporal.io/api/cloud/nexus/v1;nexus";
+
+import "google/protobuf/timestamp.proto";
+
+message IncomingServiceSpec {
+    // The name of the incoming service. Must be unique within an account
+    // The name must match `[a-zA-Z_][a-zA-Z0-9_]*`.
+    string name = 1;
+
+    // The namespace to route requests to
```

Comment:
Can we add a comment for the spec fields about their mutability.

Reply:
will do. all of these are mutable